src/main/java/org/springframework/integration/kafka/outbound/KafkaProducerMessageHandler.java

```diff
@@ -67,7 +67,9 @@
 import org.springframework.util.concurrent.SettableListenableFuture;
 
 /**
- * Kafka Message Handler.
+ * Kafka Message Handler; when supplied with a {@link ReplyingKafkaTemplate}
+ * the handler is used in an outbound gateway. When supplied with a simple
```

Comment:
used **as** an outbound ?

Reply:
No; an outbound gateway has 2 components - consumer and handler - hence in.

I will change to used as the handler in...